content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -7,99 +7,106 @@ weight: 40
 including resource limits, volumes, and security settings. However, the
-resource requests and limits for an Init Container are handled slightly
-differently, which are documented in [Resources](#resources) below.  Also, Init Containers do not
-support readiness probes because they must run to completion before the Pod can
-be ready.
+resource requests and limits for an init container are handled slightly
+differently, which are documented in [Resources](#resources) below.
+
+Also, init containers do not support readiness probes because they must run to
+completion before the Pod can be ready.
 
-If multiple Init Containers are specified for a Pod, those Containers are run
-one at a time in sequential order. Each must succeed before the next can run.
-When all of the Init Containers have run to completion, Kubernetes initializes
-the Pod and runs the application Containers as usual.
+If you specify multiple init containers for a Pod, Kubelet runs each init
+container sequentially. Each init container must succeed before the next can run.
+When all of the init containers have run to completion, Kubelet initializes
+the application containers for the Pod and runs them as usual.
 
-## What can Init Containers be used for?
+## What can init containers be used for?
 
-Because Init Containers have separate images from app Containers, they
+Because init containers have separate images from app containers, they
 have some advantages for start-up related code:
 
-* They can contain and run utilities that are not desirable to include in the
-  app Container image for security reasons.
-* They can contain utilities or custom code for setup that is not present in an app
+* Init containers can contain utilities or custom code for setup that is not present in an app
   image. For example, there is no need to make an image `FROM` another image just to use a tool like
   `sed`, `awk`, `python`, or `dig` during setup.
+* Init containers can contain and run utilities that are not desirable to include in the
+  app container image for security reasons.
 * The application image builder and deployer roles can work independently without
   the need to jointly build a single app image.
-* They use Linux namespaces so that they have different filesystem views from app Containers.
-  Consequently, they can be given access to Secrets that app Containers are not able to
-  access.
-* They run to completion before any app Containers start, whereas app
-  Containers run in parallel, so Init Containers provide an easy way to block or
-  delay the startup of app Containers until some set of preconditions are met.
+* Init containers can run with a different view of the filesystem than app containers in the
+  same Pod. Consequently, they can be given access to
+  {{< glossary_tooltip text="Secrets" term_id="secret" >}} that app containers are not able
+  to access.
+* Because init containes run to completion before any app containers start, this offers
```

Comment:
nit: s/containes/containers

Reply:
> Because init containers run to completion before any app containers start, init containers offer a mechanism to block or delay app container startup until a set of preconditions are met. Once preconditions are met, all of the app containers in a Pod can start in parallel.